Release 1.0.1 : Apple Website Clone

mimipeshy opened the pull request:
Changelog

- Created header section and added a background image

- Added navbar section 

- Added gradients to navbar section

- Used grid to create aside section

- Created the footer section

- Updated README file

- Fixed CSS linters

- Validated HTML code

mimipeshy committed:
added CI configuration files

mimipeshy committed:
finished clone project

mimipeshy committed:
fixed linter errors

mimipeshy committed:
Merge pull request #1 from mimipeshy/ft-navbar

finished clone project

mimipeshy committed:
updated README file

mimipeshy committed:
fixed spacing errors

mimipeshy committed:
updated README file

mimipeshy committed:
fixed spacing errors

mimipeshy committed:
Merge pull request #2 from mimipeshy/gh-pages

fixed spacing errors

mimipeshy committed:
fixed linter errors

mimipeshy committed:
updated README file and removed redudant css

mimipeshy committed:
Added new screenshot image

mimipeshy committed:
fixed linter errors

mimipeshy committed:
updated README file

sumancrest0001 reviewed (commented):
**Status: changes required
Awesome work @mimipeshy** 
You have done a fabulous job to make it a clone of the original apple site. You are very close to getting approval for this project. To get approval, we need to make the following changes.
**[Note] Optional changes are not mandatory but nice to have.**

sumancrest0001 commented on index.html:
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```
We have to fix the HTML validator error. I am using [this validator](https://validator.w3.org/#validate_by_upload)

sumancrest0001 commented on index.html:
```diff
@@ -0,0 +1,80 @@
+    <main>
+        <div class="header-section">
+            <nav class="gradient justify-center">
+                <ul class="grid cursor">
+                    <li><a href="#"><i class="fab fa-apple"></i></a></li>
```
[Optional] We can use an apple image for the logo or can increase the size of the font awesome icon.

sumancrest0001 commented:
[Optional] We can center the flag image and remove the vertical line before the **Site map** footer-item in the footer section to give it a similar look.

sumancrest0001 reviewed (changes requested):
Kindly make the requested change and request for another review.

Have a question or need support? DM me on slack (**Suman Shrestha**)
**Happy coding !!!**

mimipeshy committed:
fixed TSE review

mimipeshy commented:
Worked on the following review

-  Fixed HTML errors with attached screenshot. 
![htmlvalidation](https://user-images.githubusercontent.com/33067256/82219921-a38daf00-9926-11ea-8460-54f0011449c1.PNG)

I validated using this [link](https://validator.w3.org/#validate_by_input)

- Centered US logo image

- Removed vertical line before sitemap

- Increased apple logo font size

mimipeshy committed:
removed redudant css

mimipeshy committed:
changed HTML semantics

mimipeshy committed:
fixed css error

christianotieno reviewed (approved):
# PROJECT APPROVED :champagne:
Excellent job on this project. :fireworks:  Keep it up.
##
### The [clone](https://raw.githack.com/mimipeshy/backgrounds-and-gradients/gh-pages/index.html) shows good understanding and usage of building with background and gradients. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
-  The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @mimipeshy! :+1:
- You can now merge to master and I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
